sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwins_client.py

```diff
@@ -0,0 +1,547 @@
+        :rtype: object
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twins.add_relationship(
+            id=digital_twin_id,
+            relationship_id=relationship_id,
+            if_none_match="",
+            relationship=relationship,
+            **kwargs
+        )
+
+    def update_relationship(self, digital_twin_id, relationship_id, relationship_patch=None, etag=None, **kwargs):
+        # type: (str, str, List[object], str, **Any) -> object
+        """Updates the properties of a relationship on a digital twin using a JSON patch.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is either no digital twin or relationship with the provided id.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param relationship_id: The Id of the relationship to retrieve.
+        :type str
+        :param relationship_patch: JSON Patch description of the update to the relationship properties.
+        :type List[object]
+        :param etag: Only perform the operation if the entity's etag matches one of
+            the etags provided or * is provided.
+        :type str
+        :keyword str content_type: Content type to set in the HTTP header. Default to "application/json".
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twins.update_relationship(
+            id=digital_twin_id,
+            relationship_id=relationship_id,
+            relationship_patch=relationship_patch,
+            if_match=etag,
+            **kwargs
+        )
+
+    def delete_relationship(self, digital_twin_id, relationship_id, etag=None, **kwargs):
+        # type: (str, str, str, **Any) -> object
+        """Delete a digital twin.
+        Status codes:
+        200 (OK): Success.
+        404 (Not Found): There is either no digital twin or relationship with the provided id.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param relationship_id: The Id of the relationship to delete.
+        :type relationship_id: str
+        :param etag: Only perform the operation if the entity's etag matches one of
+            the etags provided or * is provided.
+        :type etag: str
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twins.delete_relationship(
+            digital_twin_id, 
+            relationship_id, 
+            if_match=etag, 
+            **kwargs
+        )
+
+    def list_relationships(self, digital_twin_id, relationship_id=None, **kwargs):
+        # type: (str, str, **Any) -> Iterable["models.RelationshipCollection"]
```

Comment:
We can document this as `ItemPaged[models.RelationshipCollection]`

Reply:
Actually RelationshipCollection shouldn't be exposed, rather it would be something like `ItemPaged[Relationship]`

Reply:
Fixed